MultiplayerTestRunner/src/InternalUnitTests/Test_SpaceRAII.cpp

```diff
@@ -0,0 +1,157 @@
+#include "../../include/ErrorCodes.h"
+#include "../../include/ProcessDescriptors.h"
+#include "../LoginRAII.h"
+#include "../SpaceRAII.h"
+#include "../Utils.h"
+
+#include <CSP/Common/String.h>
+#include <CSP/Systems/Spaces/SpaceSystem.h>
+#include <CSP/Systems/SystemsManager.h>
+#include <gtest/gtest.h>
+#include <optional>
+
+namespace
+{
+/* Some tests only run if there's a credentials file */
+std::optional<Utils::TestAccountCredentials> CredentialsFromFile()
```

Comment:
[consider] Maybe this should be a utils method, it's duplicated more than twice.

Reply:
Na, I don't think having test specific methods in Utils is that great. There's already a little bit of that sneaking in with public/private access :(

Reply:
I just love this. (sorry)